scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# This script runs the integration tests. It starts the test environment, runs the tests,
+# saves the container logs to a file, and stops the containers.
+# Usage:
+#   ./script.sh            # Run the script with default options
+#   ./script.sh --build    # Re-build the docker images (-b or --build)
+
+timeout=30
+log_file="/tmp/rafiki_integration_logs.txt"
```

Comment:
Container logs get saved here for each run. Useful for developing tests but should not normally need.

I could change to local tmp folder instead but didn't have much preference.

Reply:
I think local `tmp` folder is better IMO, then I can just see it right there.

Also, I think we can move this script inside `test/integration`?

Reply:
yeah makes sense... easier to have it locally i guess. changed to a test/integration/tmp and moved inside test/integration with a package.json script to start it